sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Custom/AppConfigurationKeyValueCollection.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Threading;
+
+namespace Azure.ResourceManager.AppConfiguration
+{
+    public partial class AppConfigurationKeyValueCollection : IEnumerable<AppConfigurationKeyValueResource>, IAsyncEnumerable<AppConfigurationKeyValueResource> // previously this class implemented these interfaces, and it must implement them now (and forever)
```

Comment:
well this comment is I explaining the purpose of this, we do not have to include this in code